platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/system/transaction/internal/TransactionSerializationTest.java

```diff
@@ -88,69 +83,6 @@ void SignatureSerializeDeserializeTest(final int sigSize) throws IOException {
         TestExpectedSerializationLength(deserialized, false);
     }
 
-    @ParameterizedTest
-    @ValueSource(ints = {0, 1, 2, 10, 64, 100})
-    void BitsSecondSerializeDeserializeTest(final int nodeAmount) throws IOException {
-        long[] numbers = null;
-        if (nodeAmount > 0) {
-            numbers = new long[nodeAmount];
-            for (int i = 0; i < numbers.length; i++) {
-                numbers[i] = random.nextLong();
-            }
-        }
-
-        final SystemTransactionBitsPerSecond systemTransactionBitsPerSecond =
-                new SystemTransactionBitsPerSecond(numbers);
-        final SystemTransactionBitsPerSecond deserialized = serializeDeserialize(systemTransactionBitsPerSecond);
-
-        assertArrayEquals(systemTransactionBitsPerSecond.getAvgBitsPerSecSent(), deserialized.getAvgBitsPerSecSent());
-        assertEquals(systemTransactionBitsPerSecond.isSystem(), deserialized.isSystem());
-        assertEquals(systemTransactionBitsPerSecond.getVersion(), deserialized.getVersion());
-        assertEquals(systemTransactionBitsPerSecond.getClassId(), deserialized.getClassId());
-        assertEquals(
-                systemTransactionBitsPerSecond.getMinimumSupportedVersion(), deserialized.getMinimumSupportedVersion());
-
-        assertEquals(systemTransactionBitsPerSecond, deserialized);
-        assertEquals(systemTransactionBitsPerSecond.getType(), SYS_TRANS_BITS_PER_SECOND);
-        assertEquals(SYS_TRANS_BITS_PER_SECOND, deserialized.getType());
-
-        TestExpectedSerializationLength(systemTransactionBitsPerSecond, true);
-        TestExpectedSerializationLength(deserialized, true);
-
-        TestExpectedSerializationLength(systemTransactionBitsPerSecond, false);
-        TestExpectedSerializationLength(deserialized, false);
-    }
-
-    @ParameterizedTest
-    @ValueSource(ints = {0, 1, 2, 10, 64, 100})
-    void PingSerializeDeserializeTest(final int nodeAmount) throws IOException {
```

Comment:
Why do you remove that tests?

Reply:
They are testing serialization for transaction types we don't use anywhere, and have no plans on using anywhere. I deleted both the transactions and the tests on the transactions. They were using static configuration, and my goal was to eliminate static config use by any means possible.